theano/configdefaults.py

```diff
@@ -443,7 +443,7 @@ def f(suffix=suffix):
 if param and os.name == 'nt':
     param = '"%s"' % param
 AddConfigVar('cxx',
-             "The C++ compiler to use. Currently only g++ is"
+             "The C++ compiler to use. Currently only g++ and icpc is"
```

Comment:
Good change. We also support clang++ as it support the same interface as g++. Can you change it to this at the same time
"only g++ (clang++) and icpc"